src/client/components/PasswordForm.tsx

```diff
@@ -64,6 +64,10 @@ const form = css`
   padding-top: ${space[3]}px;
 `;
 
+const passwordInput = css`
+  margin-bottom: ${space[3]}px;
+`;
```

Comment:
Adding the bottom margin to the password form, as it has been removed from the password input field (see below for details)